test/viewer.jl

```diff
@@ -95,7 +95,16 @@ end
     frames = Javis.preprocess_frames!(objects)
 
     @test v.filename === "foo.png"
-    img = Javis._pluto_viewer(vid, length(frames), objects)
+    img = Javis._pluto_viewer(vid, length(frames)-1, objects)
```

Comment:
does this mean that the pluto viewer will generally have problems? Like switching from the end frame of morphing back?
Maybe we should save the original function in the `Object` struct as well then and set it if it's not the last frame?

Reply:
Yeah. I breaks jupyter and javis viewer. Pluto will work since we iterate over a list of frames.